.github/workflows/ci.yml

```diff
@@ -21,12 +21,23 @@ jobs:
           - ROS_DISTRO: kinetic
             UPSTREAM_WORKSPACE: .ci.rosinstall
             ROS_REPO: main
+            DOCKER_RUN_OPTS: --network bridge
+            BEFORE_INIT: 'apt-get update -qq && apt-get install -y iproute2 iputils-ping && ip addr && ping -c5 172.17.0.2'
+            IMMEDIATE_TEST_OUTPUT: true
+            BEFORE_RUN_TARGET_TEST: 'set +u; source /root/target_ws/install/setup.bash && echo $ROS_PACKAGE_PATH | sed "s/\:/\n/g" && roslaunch ur_robot_driver ur5e_bringup.launch robot_ip:=172.17.0.2; set -u'
           - ROS_DISTRO: melodic
             UPSTREAM_WORKSPACE: .ci.rosinstall
             ROS_REPO: main
+            DOCKER_RUN_OPTS: --network bridge
+            BEFORE_INIT: 'apt-get update -qq && apt-get install -y iproute2 iputils-ping && ip addr && ping -c5 172.17.0.2'
+            IMMEDIATE_TEST_OUTPUT: true
+            BEFORE_RUN_TARGET_TEST: 'set +u; source /root/target_ws/install/setup.bash && echo $ROS_PACKAGE_PATH | sed "s/\:/\n/g" && roslaunch ur_robot_driver ur5e_bringup.launch robot_ip:=172.17.0.2; set -u'
```

Comment:
This is still an open issue. This step is currently included, as the first connection attempt made to the simulation fails (see [here](https://github.com/UniversalRobots/Universal_Robots_ROS_Driver/runs/810450510?check_suite_focus=true#step:4:1213). 

I added this step as a temporary debug step to go on with testing. We'll have to clarify where that comes from and get rid of this workaround.

Reply:
Resolved in ae75566